### DISCUSSION TITLE
Métodos Anônimos

### DISCUSSION THREAD

**Discussion by DrenokBR** [balta-io/forum#424] [Tira Dúvidas] [answered]
Procurei por alguma aula onde você explicasse "Métodos Anônimos", mas não encontrei. Não entendi este conceito.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @DecoPRG, métodos ou tipos anônimos?
👉 https://balta.io/blog/csharp-tipos-anonimos

**Reply by DrenokBR** [reply]
Fala André ! Primeiramente, muito obrigado pela resposta.  Mas acho que me confundi. Assistindo à aula

" OneToOne - Parte 2   <   Imersão   <   *Acesso a dados com .NET, C#, Dapper e SQL Server* "
(linhas 253 a 255, no momento 2min 59 seg)

Vi você usando uma função sem nome (apenas usando parênteses) como parâmetro do método .Query<>()
Seria "Função Lâmbda" o nome desse conceito. ?!?!Tem alguma aula sua tratando desse assunto ?

**Reply by andrebaltieri** [reply]
Não! Você está correto, eu que entendi errado... Isso que mostrei são funções anônimas mesmo!
Ficou alguma dúvida sobre elas?

**Reply by DrenokBR** [reply]
Ah sim... então... acho que Eu precisava assistir uma aula sobre "Funções Anônimas". Não entendi como se usa uma função sem nome, usando apenas os parênteses. Tem alguma aula de algum curso seu onde você esteja explicando esse assunto (Funções Anônimas) ??? Esse assunto faz parte do contexto de "Delegates" ???

**Reply by andrebaltieri** [reply]
Agora não vou me recordar qual aula está... mas segue os cursos e se não encontrar eu preparo um material, beleza?

**Reply by DrenokBR** [reply]
Poxa, gratidão André. Obrigado pela atenção.
